accounts/abi/abi_test.go

```diff
@@ -711,5 +711,18 @@ func TestABI_MethodById(t *testing.T) {
 			t.Errorf("Method %v (id %v) not 'findable' by id in ABI", name, common.ToHex(m.Id()))
 		}
 	}
+	// Also test empty
+	if _, err := abi.MethodById([]byte{0x00}); err == nil {
+		t.Errorf("Expected error, too short to decode data")
+	}
+	if _, err := abi.MethodById([]byte{}); err == nil {
+		t.Errorf("Expected error, too short to decode data")
+	}
+	if _, err := abi.MethodById(nil); err == nil {
+		t.Errorf("Expected error, nil is short to decode data")
+	}
+}
+
+func TestABI_MethodByIdEmpty(t *testing.T) {
```

Comment:
I guess this is a leftover from a previous test, and that you moved the test to `TestABI_MethodById` ? Anyhow, no need to commit an empty function.